docker-compose.yaml

```diff
@@ -26,7 +26,8 @@ services:
       - postgres
 
   minio:
-    image: minio/minio:RELEASE.2023-01-02T09-40-09Z
+    image: minio/minio:RELEASE.2022-10-24T18-35-07Z.hotfix.ce525fdaf
+    #image: minio/minio:RELEASE.2023-01-02T09-40-09Z #had some issue with this ver
```

Comment:
Keep the newer version in the repository, even if in your local system you use a different one.

Reply:
By the way, what problem do you have with the new version ?
You might need to delete the docker volume.